server/utils/tz/conversion.go

```diff
@@ -0,0 +1,743 @@
+	"West Asia Standard Time":         "Asia/Tashkent",
+	"Georgian Standard Time":          "Asia/Tbilisi",
+	"Iran Standard Time":              "Asia/Tehran",
+	"Tokyo Standard Time":             "Asia/Tokyo",
+	"Tomsk Standard Time":             "Asia/Tomsk",
+	"Ulaanbaatar Standard Time":       "Asia/Ulaanbaatar",
+	"Vladivostok Standard Time":       "Asia/Vladivostok",
+	"Yakutsk Standard Time":           "Asia/Yakutsk",
+	"Ekaterinburg Standard Time":      "Asia/Yekaterinburg",
+	"Caucasus Standard Time":          "Asia/Yerevan",
+	"Azores Standard Time":            "Atlantic/Azores",
+	"Cape Verde Standard Time":        "Atlantic/Cape_Verde",
+	"Greenwich Standard Time":         "Atlantic/Reykjavik",
+	"Cen. Australia Standard Time":    "Australia/Adelaide",
+	"E. Australia Standard Time":      "Australia/Brisbane",
+	"AUS Central Standard Time":       "Australia/Darwin",
+	"Aus Central W. Standard Time":    "Australia/Eucla",
+	"Tasmania Standard Time":          "Australia/Hobart",
+	"Lord Howe Standard Time":         "Australia/Lord_Howe",
+	"W. Australia Standard Time":      "Australia/Perth",
+	"AUS Eastern Standard Time":       "Australia/Sydney",
+	"UTC":                             "UTC",
+	"Astrakhan Standard Time":         "Europe/Astrakhan",
+	"W. Europe Standard Time":         "Europe/Berlin",
+	"GTB Standard Time":               "Europe/Bucharest",
+	"Central Europe Standard Time":    "Europe/Budapest",
+	"E. Europe Standard Time":         "Europe/Chisinau",
+	"Turkey Standard Time":            "Europe/Istanbul",
+	"Kaliningrad Standard Time":       "Europe/Kaliningrad",
+	"FLE Standard Time":               "Europe/Kiev",
+	"GMT Standard Time":               "Europe/London",
+	"Belarus Standard Time":           "Europe/Minsk",
+	"Russian Standard Time":           "Europe/Moscow",
+	"Romance Standard Time":           "Europe/Paris",
+	"Russia Time Zone 3":              "Europe/Samara",
+	"Saratov Standard Time":           "Europe/Saratov",
+	"Central European Standard Time":  "Europe/Warsaw",
+	"Mauritius Standard Time":         "Indian/Mauritius",
+	"Samoa Standard Time":             "Pacific/Apia",
+	"New Zealand Standard Time":       "Pacific/Auckland",
+	"Bougainville Standard Time":      "Pacific/Bougainville",
+	"Chatham Islands Standard Time":   "Pacific/Chatham",
+	"Easter Island Standard Time":     "Pacific/Easter",
+	"Fiji Standard Time":              "Pacific/Fiji",
+	"Central Pacific Standard Time":   "Pacific/Guadalcanal",
+	"Hawaiian Standard Time":          "Pacific/Honolulu",
+	"Line Islands Standard Time":      "Pacific/Kiritimati",
+	"Marquesas Standard Time":         "Pacific/Marquesas",
+	"Norfolk Standard Time":           "Pacific/Norfolk",
+	"West Pacific Standard Time":      "Pacific/Port_Moresby",
+	"Tonga Standard Time":             "Pacific/Tongatapu",
+}
+
+// Source: https://techsupport.osisoft.com/Documentation/PI-Web-API/help/topics/timezones/iana.html
+var iANAToWindows = map[string]string{
```

Comment:
nit: if it's an exact reverse, you can in this case build it from the first map, in `init()`? Also, 1/5 the map definition should be in its own file, separate from the code. Suggest, 0/5 `data.go`.

Reply:
It's not an exact reverse. Windows -> IANA is one-to-many. I'd like to discuss the implications with you when we can get together.